js/listdates.js

```diff
@@ -1,87 +1,58 @@
 /* global moment, humanizeDuration */
 var content = "";
```

Comment:
Opened issue #8 for the IIFE idea

You raise a good point about document.ready — I'm not using jQuery, but there's a native equivalent — I guess I'm not clear if that's something I need? Is that just a sanity check in case my script somehow loaded before the page was ready?